lib/msal-core/src/AuthResponse.ts

```diff
@@ -12,9 +12,9 @@ export type AuthResponse = {
     tenantId: string;
     tokenType: string;
     idToken: IdToken;
-    accessToken: object;
+    accessToken: string;
     scopes: Array<string>;
-    expiresIn: string;
+    expiresOn: Date;
```

Comment:
did you confirm what other teams are using?

Reply:
other teams are also using expiresOn